docs/source/checks/tabular/train_test_validation/plot_whole_dataset_drift.py

```diff
@@ -8,53 +8,30 @@
 
 **Structure:**
 
-* `What is a dataset drift? <#what-is-a-dataset-drift>`__
+* `What Is Multivariate Drift? <#what-is-a-multivariate-drift>`__
 * `Loading the Data <#loading-the-data>`__
-* `Run the check <#run-the-check>`__
-* `Define a condition <#define-a-condition>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
 
-What is a dataset drift?
+What Is Multivariate Drift?
 ========================
-A whole dataset drift, or a multivariate dataset drift, occurs when the
-statistical properties of our input feature change, denoted by a change
-in the distribution P(X).
 
-Causes of data drift include:
+Drift is simply a change in the distribution of data over time, and it is
+also one of the top reasons of a machine learning model performance degrades
+over time.
 
-* Upstream process changes, such as a sensor being replaced that changes
-  the units of measurement from inches to centimeters.
-* Data quality issues, such as a broken sensor always reading 0.
-* Natural drift in the data, such as mean temperature changing with the seasons.
-* Change in relation between features, or covariate shift.
+A multivariate drift is a drift that occurs in more than one feature at a time,
+and may even affect the relationships between those features, which are undetectable by
+univariate drift methods.
 
-The difference between a :doc:`feature drift
-</checks_gallery/tabular/train_test_validation/plot_train_test_feature_drift>`
-(or univariate dataset drift) and a multivariate drift is that in the
-latter the data drift occures in more that one feature.
+For more information on drift, please visit our :doc:`drift guide </user-guide/general/drift_guide.rst>`.
```

Comment:
Relevant section?

Reply:
I can link to the part where we describe different types of drift, but I thought it would be better to link to the entire guide. Other more specific references are given later. WDYT?